File: app/src/main/java/com/rowland/loading/layouts/TransluscentLoadingLayout.java
package com.rowland.loading.layouts;

import android.content.Context;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.widget.FrameLayout;
import android.widget.RelativeLayout;

/**
 * Created by Rowland on 10/21/2017.
 */

public class TransluscentLoadingLayout extends RelativeLayout {

    public TransluscentLoadingLayout(@NonNull Context context) {
        super(context);
    }

    public TransluscentLoadingLayout(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public void onFinishInflate() {
        super.onFinishInflate();

    }

    @Override
    public void onLayout(boolean changed, int l, int t, int r, int b) {
        super.onLayout(changed, l, t, r, b);
    }
}

File: app/src/main/java/com/rowland/loading/layouts/TransluscentLoadingLayout.kt
package com.rowland.loading.layouts

import android.content.Context
import android.util.AttributeSet
import android.widget.FrameLayout
import android.widget.RelativeLayout

/**
 * Created by Rowland on 10/21/2017.
 */

class TransluscentLoadingLayout : RelativeLayout {

    constructor(context: Context) : super(context) {}

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {}

    public override fun onFinishInflate() {
        super.onFinishInflate()

    }

    public override fun onLayout(changed: Boolean, l: Int, t: Int, r: Int, b: Int) {
        super.onLayout(changed, l, t, r, b)
    }
}
